courses/olomouc.yml

```diff
@@ -0,0 +1,318 @@
+title: Začátečnický kurz Pythonu
```

Comment:
Když máme jen jeden kurz, tak místo `courses/olomouc.yml` to hoďme do `course.yml` (v root složce), zjednodušší to trochu konfiguraci na hlavním naucse repu.

Reply:
Ack